### DISCUSSION TITLE
Support for Next.js's Runtime Configuration

### DISCUSSION THREAD

**Discussion by Vinnl** [RyanClementsHax/storybook-addon-next#44] [Ideas]
My app makes use of Next.js's Runtime Configuration: https://nextjs.org/docs/api-reference/next.config.js/runtime-configuration

However, when I try to write a story for a component with a call to

```js
import getConfig from 'next/config'
```

I get an error like this:

```
next_config__WEBPACK_IMPORTED_MODULE_0__["default"] is not a function

getRuntimeConfig@http://localhost:6006/main.e4df4b61bd4bd6d79990.hot-update.js:148:65
```

![image](https://user-images.githubusercontent.com/4251/155991502-210c025b-0ee0-41e8-ae39-bdc80c7505eb.png)

It would be nice to have this Next.js feature supported too (or alternatively, to this at least be mentioned in the docs as a limitation of this plugin :) ).

**Comment by RyanClementsHax** [comment]
Thanks for bringing this up! I'd be happy to help. Can you open up an issue and provide a reproduction repo?

**Reply by Vinnl** [reply]
Sure!

Issue: https://github.com/RyanClementsHax/storybook-addon-next/issues/47

Repo: https://github.com/Vinnl/repro-storybook-addon-next-getconfig
